class.jetpack.php

```diff
@@ -7132,6 +7132,7 @@ public function wp_dashboard_setup() {
 		if ( has_action( 'jetpack_dashboard_widget' ) ) {
 			$widget_title = sprintf(
 				wp_kses(
+					/* Translators: Placeholder is a Jetpack logo. */
```

Comment:
We usually use `translators:` (~187 instances) rather than `Translators:` (~17 instances), but both [work](https://github.com/wp-cli/i18n-command/blob/feb52cc60f40ac564f5fd502212ad1b8b03c6f64/src/PhpCodeExtractor.php#L14).

Reply:
done in 0af2d9879c0e6cbb91a97acc05452001fa7dcfcf